src/main/scala/li/cil/oc/server/agent/Inventory.scala

```diff
@@ -9,9 +10,12 @@ import net.minecraft.item.Item
 import net.minecraft.item.ItemStack
 import net.minecraft.nbt.NBTTagCompound
 import net.minecraft.nbt.NBTTagList
-import li.cil.oc.util.ExtendedInventory._
+import net.minecraftforge.items.IItemHandlerModifiable
+import net.minecraftforge.items.wrapper.PlayerInvWrapper
 
 class Inventory(val agent: internal.Agent) extends InventoryPlayer(null) {
+  protected implicit def toItemHandler(inv: Inventory):IItemHandlerModifiable = new PlayerInvWrapper(inv)
```

Comment:
Change that to `new InvWrapper(inv)` instead to avoid NPEs in the player wrapper. Yes, agents are sorta-kinda fake players, but our agent inventory is just a wrapper around their regular inventory anyway, so this fine.